ui/tests/integration/components/policy-example-test.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'vault/tests/helpers';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+const SELECTORS = {
+  policyText: '[data-test-modal-title]',
+  policyDescription: (type) => `[data-test-example-modal-text=${type}]`,
+  jsonText: '[data-test-example-modal-json-text]',
+  informationLink: '[data-test-example-modal-information-link]',
+};
+
+module('Integration | Component | policy-example', function (hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(function () {
+    this.store = this.owner.lookup('service:store');
+  });
+
+  test('it renders the correct paragraph for ACL policy', async function (assert) {
+    this.model = this.store.createRecord('policy/acl');
+
+    await render(hbs`
+    <PolicyExample
+      @policyType={{this.model.policyType}}
```

Comment:
The `@policyType` parameter is just a string, so we don't actually have to go through the trouble of creating a model record for each of these tests. Passing in the policy type is sufficient to test the component is displaying what we expect: 

```suggestion
      @policyType="acl"
```

Cleaning this up means we can also remove the `beforeEach` hook since we'll no longer need access to the `store`. 😄